patch/base/subscriptions/patch-operator/subscription.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: patch-operator
+  namespace: patch-operator
+spec:
+  channel: alpha
+  installPlanApproval: Automatic
+  name: patch-operator
+  source: community-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: patch-operator.v0.1.9
```

Comment:
Should we omit this startingCSV since we are using automatic installPlan? In Operate First we typically don't include startingCSV with installPlanApproval: Automatic

Reply:
Good catch @dystewart , I have removed the startingCSV.